platformio/lib/OneWire/OneWire.cpp

```diff
@@ -0,0 +1,62 @@
+#include "OneWire.h"
+
+void OneWire::reset()
+{
+    _pin.output();
+    _pin = 0;
+    wait_us(500);
+    _pin.input();
+    wait_us(500);
+}
+
+void OneWire::write_bit(bool bit)
+{
+    if (bit) {
+        _pin.output();
+        _pin = 0;
+        wait_us(5);
+        _pin.input();
+        wait_us(55);
+    } else {
+        _pin.output();
+        _pin = 0;
+        wait_us(60);
+        _pin.input();
+    }
+}
+
+// output byte d (least sig bit first).
+void OneWire::write_byte(unsigned char d)
```

Comment:
In the header file, the parameter `d` is declared as `uint8_t`, not `unsigned char`.